### PR TITLE
Ensure text not beneath icons

### PR DESCRIPTION
**Pull request by waracci**
Use 80% width

**Before**

Screenshot from 2022-02-08 23-39-15

**After**

Screenshot from 2022-02-08 23-38-47

### CODE REVIEW

File: pombola/south_africa/static/sass/_south-africa_overrides.scss [hunk 1]
```diff
@@ -1173,6 +1173,9 @@ $logo-width-large: 184px;
                     line-height: 1.4;
                     letter-spacing: 0.01em;
                     width: 95%;
+                    @media screen and (max-width: 479px) {
+                        max-width: 80%;
+                    }
```
Comment (jbothma) [inline pombola/south_africa/static/sass/_south-africa_overrides.scss:1178]: I don't think it should apply to all buttons - only buttons with icons on the right.

File: pombola/south_africa/static/sass/_south-africa_overrides.scss [hunk 2]
```diff
@@ -1173,6 +1173,9 @@ $logo-width-large: 184px;
                     line-height: 1.4;
                     letter-spacing: 0.01em;
                     width: 95%;
+                    @media screen and (max-width: 479px) {
+                        max-width: 80%;
```
Comment (jbothma) [inline pombola/south_africa/static/sass/_south-africa_overrides.scss:1177]: I don't think a %-based approach is correct - the thing on the right is fixed-width, which means we need to give it the space it needs, not a variable amount of space.

Comment (waracci) [reply pombola/south_africa/static/sass/_south-africa_overrides.scss:1177]: noted, declining PR. I'll create another one.

File: pombola/south_africa/static/sass/_south-africa_overrides.scss (hunk 1 above)
Comment (waracci) [reply pombola/south_africa/static/sass/_south-africa_overrides.scss:1178]: noted, declining PR. I'll create another one.

**Pull request closed without merging**